CONTRIBUTING.md

```diff
@@ -126,6 +126,19 @@ branch that is checked out in a separate worktree.
     git log --stat -p aaaaaa...bbbbbb
     ```
 
+## Polish a change
+
+Prior to pushing your change, remember to run clang-format on the `c/cpp` files, clang-format workflow may fail otherwise. You can do so by running the following command.
+
+`find . -regex '.*\.\(c\|cpp\|h\|hpp\)' | sort | xargs clang-format --verbose -i`.
```

Comment:
This is the exact command from the workflow, but I am wondering if it makes sense to shorten it, e.g.,
```
find . -regex '.*\.\(c\|cpp\|h\)' -exec clang-format -i {} +
```
In the workflow the verbose listing of processed files does not disturb when not needed, but it does when manually used on the console. We don't actually have `hpp` files, maybe that should be removed from the workflow as well.

Nitpick: You can use a Markdown code block here:
~~~
```
code
```
~~~

Reply:
I gave it a try, and it seems like that will result in some version mismatch error.

Reply:
What error are you getting for the `-exec` variant?

In any case, how about adding a one-liner script for the longer clang-format script, e.g.,
```
find . -regex '.*\.\(c\|cpp\|h\)' | sort | xargs clang-format -i "$@"
```
which allows passing `--verbose` as an optional argument to list files if desired.

Reply:
It was an warning about version not found `clang-format: /lib64/libtinfo.so.5: no version information available (required by clang-format)` but overall the files are still getting formatted. I have changed the command to the one-liner you proposed.

Reply:
> how about adding a one-liner script for the longer clang-format script

Its a good idea. In this case, should we add for both linux and windows?

Reply:
On Windows we require `cygwin` (which is used in some of the unit tests), so a bash script only should be fine.

Reply:
Got it, I have added the bash script.